recipe/meta.yaml

```diff
@@ -6,7 +6,7 @@ package:
   version: 3.3
 
 build:
-  number: 0
+  number: 181211
```

Comment:
why are you setting this to 181211?

Reply:
This is a meta project, where I expect to need to update the build number to account for issues like conda-forge rerendering.  Other than that, there will likely be few "revision" changes to this project.

Given this, it seems semantically clearer to use a build number that indicates the date of the last update:  YYMMDD.